src/views/FindOrder.vue

```diff
@@ -254,6 +259,9 @@ export default defineComponent ({
     }
   },
   methods: {
+    async closeFilters() {
```

Comment:
Why not name method closeMenu so that it can be used on any menu that we want?

Reply:
Ok Sir, I have renamed this method to closeMenu.